java/vector/src/test/java/org/apache/arrow/vector/TestListVector.java

```diff
@@ -454,7 +454,7 @@ public void testNestedListVector() throws Exception {
       /* allocate memory */
       listWriter.allocate();
 
-      /* the dataVector that backs a listVector will also be a
+      /* the dataBuffer that backs a listVector will also be a
```

Comment:
Maybe some of these code comment changes aren't quite right?

Reply:
These are likely done be the IDE, I will double check.

Reply:
Fixed here https://github.com/wesm/arrow/pull/3